docs/src/main/asciidoc/native-and-ssl-guide.adoc

```diff
@@ -178,29 +181,16 @@ Now, the application should work as expected:
  * you should see a JSON output with some information about Greece
  * hit `Ctrl+C` to stop the application
 
-[TIP]
-====
-The root certificates file of GraalVM might not be totally up to date.
-If you have issues with some certificates, your best bet is to include the `cacerts` file of a regular JDK instead.
-====
-
-[WARNING]
-====
-Don't forget to move your GraalVM directory back to where it was.
-====
-
-=== Working with containers
-
 When working with containers, the idea is to bundle both the SunEC library and the certificates in the container and to point your binary to them using the system properties mentioned above.
 
-You can for example modify your `Dockerfile.native` as follows to copy the required files to your final image:
+You can for example modify your `Dockerfile.native` as following to allow copying the required lib in your final image:
```

Comment:
Looks like you reverted some changes I made recently.

Reply:
Good catch. Sorry for the noise. I'll rebase and apply the change I need to make.